packages/starlight/translations/index.ts

```diff
@@ -15,7 +15,7 @@ import da from './da.json';
 import tr from './tr.json';
 import ar from './ar.json';
 import nb from './nb.json';
-import zh from './zh.json';
+import zhCN from './zh-CN.json';
```

Comment:
As @Genteure suggested, we can probably stick with this being `zh` here. That will mean people already using `lang: 'zh'` can keep using this. And our system is able to pick this up anyway, so sites using `lang: 'zh-CN'` will also get this file.

```suggestion
import zh from './zh-CN.json';
```